Extension/ui/settings.html

```diff
@@ -499,7 +499,9 @@
         <div class="section-title">IntelliSense mode</div>
         <div class="section-text">
             The IntelliSense mode used by the IntelliSense engine. <code>msvc-x64</code> maps to Visual Studio mode with 64-bit pointer sizes. 
-            <code>clang-x64</code> maps to CLang mode with 64-bit pointer sizes. <code>gcc-x64</code> maps to GCC mode with 64-bit pointer sizes. 
+            <code>clang-x64</code> maps to CLang mode with 64-bit pointer sizes. <code>gcc-x64</code> maps to GCC mode with 64-bit pointer sizes.

```

Comment:
Let's simplify description of `IntelliSense` to:

> The IntelliSense mode to use that maps to an architecture-specific variant of MSVC, gcc, or Clang. If not set or if set to `${default}`, the extension will choose the default for that platform. Windows defaults to `msvc-x64`, Linux defaults to `gcc-x64`, and macOS defaults to `clang-x64`. 

Need to also update https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference

Reply:
Got it